health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/CampaignBoundary.js

```
@@ -0,0 +1,798 @@
+            "parent": "MICROPLAN_MO_04_GBARPOLU",
+            "isRoot": false,
+            "includeAllChildren": false
+        }
+    ]
+
+
+
+    // Memoizing parents and parent_group to only compute once when selectedData changes
+
+    const { parents, parent_group } = useMemo(() => {
+        let parents = {};
```

Biome

[error] This let declares a variable that is only assigned once.

'parents' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)